lambdaNotes/src/NotesPage.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import { Route } from "react-router-dom";
+import Login from "./components/Login";
+import axios from "axios";
+import NotesList from "./components/NotesList";
+import Note from "./components/Note.js";
+import EditNote from "./components/EditNote";
+import AddNote from "./components/AddNote";
+import { Container } from "./Styles";
+import "./App.css";
+
+class NotesPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      notes: []
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get("https://fe-notes.herokuapp.com/note/get/all")
+      .then(response => this.setState({ notes: response.data }))
+      .catch(error => console.log(error));
+  }
+
+  updateNotes = updatedNote => {
+    console.log(updatedNote);
+    const updatedNotes = this.state.notes.map(note => {
+      if (note._id === updatedNote._id) {
+        return updatedNote;
+      }
+      return note;
+    });
+    this.setState({ notes: updatedNotes });
+  };
+
+  updateDeleted = targetID => {
+    console.log(targetID);
+    const updatedLists = this.state.notes.filter(note => {
```

Comment:
you could simplify this further with:
```suggestion
    const updatedLists = this.state.notes.filter(note => note._id === targetID);
```